docs/docs/policies.mdx

```diff
@@ -321,11 +340,11 @@ data as well.
 ### Rule Policy
 
 The `RulePolicy` is a policy that handles conversation parts that follow
-a fixed behavior. It makes predictions based on any `rules` you have in your
-training data. See the [Rules documentation](./rules.mdx) for further information
-on how to define rules.
+a fixed behavior (e.g. business logic). It makes predictions based on
+any `rules` you have in your training data. See the
+[Rules documentation](./rules.mdx) for further information on how to define rules.
 
-The RulePolicy has the following configuration options:
+The `RulePolicy` has the following configuration options:
```

Comment:
I unified with above, but I think we never want backticks, right?

Reply:
really? I think we want to code-ify anything that is code, and i would think the names of policies would be the same. I'm not sure though. @karen-white  what do you think?

Reply:
the reason I wrote the comment is because I looked at the history of the file and saw [this commit](https://github.com/RasaHQ/rasa/commit/e72ed84c12a2771835eedbe7efba84d3d6036fc3)

Reply:
I added this to a list of things we need to clarify in the style guide 👍